controllers/datadogagent/override/global.go

```diff
@@ -137,6 +137,13 @@ func applyGlobalSettings(logger logr.Logger, manager feature.PodTemplateManagers
 		}
 	}
 
+	if config.OriginDetectionUnified != nil && config.OriginDetectionUnified.Enabled != nil {
```

Comment:
Could you update the corresponding test in this file https://github.com/DataDog/datadog-operator/blob/4838b3cae44ea0329ba036b9f25a8fe4a6cf7f2e/controllers/datadogagent/override/global_test.go#L34

Reply:
I've added a case 👍